Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1708,6 +1708,10 @@ extension BrowserViewController: IntroViewControllerDelegate {
     }
 
     func introViewControllerDidFinish(_ introViewController: IntroViewController) {
+        if self.profile.prefs.intForKey(PrefsKeys.IntroSeen) == nil, let url = URL(string: Strings.WhatsNewWebsite) {
```

Comment:
Ok so I think intro and whats new should be two separate things. Can you please have separate prefs Keys for `WhatsNewSeen` and `IntroSeen?

Then in `viewDidAppear` you can first check for IntroSeen and if not, show the Intro. And also check of `WhatsNewSeen` and if not, open the what's new page. The what's new tab should load behind the intro automatically I think.

Reply:
I have tried it, there is an issue. In viewDidAppear what's new page doesn't load.